ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -77,6 +77,16 @@ export class PostCommand extends IronfishCommand {
 
     const posted = new Transaction(Buffer.from(response.content.transaction, 'hex'))
 
+    if (response.content.accepted === false) {
+      this.log(
+        `Transaction '${posted.hash().toString('hex')}' was not accepted into the mempool`,
+      )
+    }
+
+    if (response.content.broadcasted === false) {
+      this.log(`Transaction '${posted.hash().toString('hex')}' failed to broadcast`)
+    }
+
```

Comment:
I think we'll want this logic in the send, mint, and burn commands, too

Reply:
Yep, I was gonna do in a subsequent PR but will add here.